di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -341,9 +344,25 @@ public AuthorizeHandler(
                             queryParamsMap
                                     .value(CredentialIssuerConfig.EVIDENCE_CONTRAINDICATOR_PARAM)
                                     .replaceAll("\\s", "");
+                    String encodedUserId;
+                    try {
+                        encodedUserId =
+                                URLEncoder.encode(userId, StandardCharsets.UTF_8.toString());
+                    } catch (UnsupportedEncodingException e) {
+                        throw new CriStubException("Unable to URL encode userId", e);
+                    }
+
                     if (!ciString.isEmpty()) {
                         String[] ciList = ciString.split(",");
-                        gpgMap.put(CredentialIssuerConfig.EVIDENCE_CONTRAINDICATOR_PARAM, ciList);
+                        if (!ciString.isEmpty()) {
+                            String iss =
+                                    signedJWT
+                                            .getJWTClaimsSet()
+                                            .getClaim(RequestParamConstants.ISSUER)
+                                            .toString();
+                            processIssuersCIs(
```

Comment:
I'm a bit confused about this - are you adding an extra call to the stub management API that wasn't here before?

I don't think you need to do this - we should be setting the CIs/Issuers when IPV Core submits the VC to the CIMIT stub (i.e. in `ContraIndicatorsService`). I think this change will result in duplicate CIs being recorded?